R/uploadPackage.R

```diff
@@ -125,6 +139,11 @@ uploadPackageServer <- function(id) {
               
               uploaded_packages$status[i] <- 'not found'
               
+              # Suggest alternative spellings using utils::adist() function
+              v <- utils::adist(uploaded_packages$package[i], CRAN_arch, ignore.case = FALSE)
+              rlang::inform(paste("Package name",uploaded_packages$package[i],"was not found."))
+              rlang::inform(paste("suggested package names:",paste(head(CRAN_arch[which(v == min(v))], 10),collapse = ", ")))
```

Comment:
Is there a reason we are sending this information to the console instead of the UI?

Also, should we expand the suggested packages from just 1 to maybe 5?

Reply:
I guess I wasn't sure where to put this information, the console was the simplest place to put it
![image](https://user-images.githubusercontent.com/75215958/196774240-5829c73b-e680-4865-829a-416e4e5ee71b.png)

And the `head(...,10)` function will provide up to ten alternative suggestions, if it can find them.